### PR TITLE
Adding Sqs Producer Group Id & Deduplication Id

### PR DESCRIPTION
**Pull request by forrestbice**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/bbc/sqs-producer#usage
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/sqs-producer/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 // Type definitions for sqs-producer 1.5
 // Project: https://github.com/BBC/sqs-producer
 // Definitions by: Daniel Chao <http://dchao.co/>
+                   Forrest Bice <https://github.com/forrestbice>
```
Comment (andy-ms) [inline types/sqs-producer/index.d.ts]: Needs `//` (or it's not a comment)

Comment (forrestbice) [reply types/sqs-producer/index.d.ts]: @andy-ms whoops! Fixed now

**Comment by andy-ms** [comment]
Looks like there are a few test failures:
```
ERROR: 64:4   only-arrow-functions  non-arrow functions are forbidden
ERROR: 61:3   expect                TypeScript@next compile error: 
Argument of type '{ body: string; groupId: string; deduplicationId: string; }' is not assignable to parameter of type 'string[] | ProducerMessage[]'.
  Object literal may only specify known properties, and 'body' does not exist in type 'string[] | ProducerMessage[]'.
ERROR: 64:13  expect                TypeScript@next compile error: 
Parameter 'err' implicitly has an 'any' type.
```

**Comment by RyanCavanaugh** [comment]
@forrestbice Please fix the failures indicated in the Travis CI log.

**Comment by minestarks** [comment]
I'm closing this PR due to test failures that haven't been addressed in over a month. Please open a new PR if you would still like to get these changes in. Thank you!

**Pull request closed without merging**